os/fs/procfs/fs_procfsproc.c

```diff
@@ -103,8 +104,12 @@
  * accessed via the procfs file system.
  */
 
+/* '/proc/[pid]/stat' and '/proc/[pid]/status' entries are almost similar, they provide information about task/pthread.
+ * The 'status' entry is easier for humans to parse.
+ */
 enum proc_node_e {
 	PROC_LEVEL0 = 0,			/* The top-level directory */
+	PROC_STAT,					/* Task/thread status vales */
```

Comment:
1. Let's align comments.
2. What is a WORD difference between `STAT` and `STATUS`?
3. What is a difference between `status values` and `status`?

Reply:
We support 2 kinds of entry to provide information of task/pthread.
The difference is a form of printed string only.
The 'status' entry is easier for human to parse.

Reply:
I don't understand. Please let me clarify this difference.

Reply:
The 'stat' provides system information values only.
On the other hand, 'status' shows these values with prefix about info in details.